stix_shifter_modules/cybereason/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,2299 @@
+        "object": "user_email",
+        "references": "user"
+      }
+    ],
+    "isAdmin": {
+      "key": "user-account.extensions.x-cybereason-user.is_admin",
+      "object": "user"
+    },
+    "adCreated": [
+      {
+        "key": "user-account.account_created",
+        "object": "user",
+        "transformer": "EpochToTimestamp"
+      },
+      {
+        "key": "first_observed",
+        "cybox": false,
+        "transformer": "EpochToTimestamp"
+      },
+      {
+        "key": "last_observed",
+        "cybox": false,
+        "transformer": "EpochToTimestamp"
+      }
+    ],
+    "domain": {
+      "key": "user-account.extensions.x-cybereason-user.domain",
+      "object": "user"
+    },
+    "ownerOrganization.name": {
+      "key": "user-account.extensions.x-cybereason-user.organizational_unit",
+      "object": "user"
+    },
+    "ownerMachine": {
+      "key": "user-account.extensions.x-cybereason-user.last_logged_machine",
+      "object": "user"
+    },
+    "isLocalSystem": {
+      "key": "user-account.extensions.x-cybereason-user.is_local_system",
+      "object": "user"
+    },
+    "adDepartment": {
+      "key": "user-account.extensions.x-cybereason-user.department",
+      "object": "user"
+    },
+    "adCountry": {
+      "key": "user-account.extensions.x-cybereason-user.company",
+      "object": "user"
+    },
+    "adCompany": {
+      "key": "user-account.extensions.x-cybereason-user.company",
+      "object": "user"
+    },
+    "adAssociatedDomain": [
+      {
+        "key": "domain-name.value",
+        "object": "user_domain"
+      },
+      {
+        "key": "user-account.extensions.x-cybereason-user.associated_domain",
+        "object": "user"
+      },
+      {
+        "key": "user-account.extensions.x-cybereason-user.associated_domain",
+        "object": "user",
+        "references": "user_domain"
+      }
+    ],
+    "emailAddress": [
+      {
+        "key": "user-account.extensions.x-cybereason-user.email_address",
+        "object": "user"
+      },
+      {
+        "key": "email-addr.value",
+        "object": "user_email"
+      }
+    ],
+    "numberOfMachines": [
+      {
+      "key": "user-account.extensions.x-cybereason-user.total_machines",
+      "object": "user"
+      },
+      {
+        "key": "autonomous-system.number",
+        "object": "user-as",
+        "references": "ToInteger"
+      }
+    ],
+    "passwordAgeDays": {
+      "key": "user-account.extensions.x-cybereason-user.password_age_days",
+      "object": "user"
+    },
+    "privileges": {
+      "key": "user-account.extensions.x-cybereason-user.privileges",
+      "object": "user"
+    },
+    "comment": {
+      "key": "user-account.extensions.x-cybereason-user.comment",
+      "object": "user"
+    },
+    "adCanonicalName": {
+      "key": "x-oca-asset.host_name",
+      "object": "asset"
+    },
+    "adMail": [
+      {
+        "key": "user-account.extensions.x-cybereason-user.mail",
```

Comment:
This should be an `email-addr` object instead.

Reply:
Changes has been done as per the suggestion  in to_stix_map.json file.